ishell.go

```diff
@@ -60,6 +60,7 @@ type Shell struct {
 	pagerArgs         []string
 	isUninterpreted   bool
 	lineTerminator    string
```

Comment:
Does it still make sense to maintain this variable if lineTerminators exists now? The Get/Set api for this variable may no longer make sense.

Also I'm slightly skeptical that the ishell library does not support multiple terminator options. Can you confirm in their github issues page this is correct?

Reply:
i'm using separate `lineTerminator` for setting it to which of the `;`, `\g` or `\G` was used, so that with Get function, in dolt i use it to determine the returnFormat and trim the line terminator from the query if necessary.
With Set function, in dolt i use it to update the delimiter `;` to anything else, so that the new delimiter is used instead of `;`.

There is no line terminator is used in the original ishell library, and there isn't anything in the issues as well.

Reply:
I see. Please rename `lineTerminators` to `lineTerminatorOptions` so it is more clear

Reply:
got it